src/transformStateWithClones.js

```diff
@@ -65,7 +65,37 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const array = [];
```

Comment:
```suggestion
  const stateClones = [];
```
Переименуй переменную, чтоб было понятнее что в ней может находится.